data/wingpanel.appdata.xml.in

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-2019 elementary, Inc. <contact@elementary.io> -->
+<component type="desktop">
+  <id>io.elementary.desktop.wingpanel</id>
```

Comment:
```suggestion
  <id>io.elementary.wingpanel</id>
```
We don't need that level of namespacing (because then the indicators are even deeper…)

Reply:
I was trying to be consistent with what we already have for the gsettings keys, but I agree that this is a lot.

Would it make sense to redo everything as `io.elementary.panel`?

Reply:
Whatever makes the more sense, io.elementary.panel would work